JetStreamDriver.js

```diff
@@ -2359,21 +2324,51 @@ function processTestList(testList)
     for (let name of benchmarkNames) {
         name = name.toLowerCase();
```

Comment:
nit: we `toLowerCase()` already inside `enableBenchmarksByTag` and `enableBenchmarkByName` below, so this could be just removed here (or there).